### DISCUSSION TITLE
Question about the system cost calculation

### DISCUSSION THREAD

**Discussion by wanyingw193** [NREL/ReEDS-2.0#88] [Q&A: Data]
Hi ReEDS team, thank you for your attention!

I compared the runs/[casename]/outputs/systemcost.csv to the bokehpivot/[casename]/result-Sys Cost Objective (bil $), presets-Cost by year. 

In the bokehpivot/reeds2.py, per_systemcost function say the 
"if 'objective' in kw and kw['objective'] == True:
        #Multiply all capital costs by pvf_capital and operation by pvf_onm"

However, after multiplying by pvf_capital/pvf_onm, I cannot get the same number in the bokehpivot Html. Am I going in the wrong direction? Is there any documentation that explains the system cost calculation in reeds2.py? Thank you so much?

**Comment by wesleyjcole** [comment]
The system cost calculations in bokepivot use a pre_systemcost function (see https://github.com/NREL/ReEDS_OpenAccess/blob/master/bokehpivot/reeds2.py#L1750).  You can walk through that function to see what calculations are taking place in bokehpivot.  This portion of code will also show you how the presets are configured, which might result in certain categories getting excluded in bokehpivot.
